FreetimeTests/SortingMentionedByDayTests.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SortingMentionedByDayTests.swift
+//  FreetimeTests
+//
+//  Created by Alexey Karataev on 09.03.2019.
+//  Copyright © 2019 Ryan Nystrom. All rights reserved.
+//
+
+import XCTest
+@testable import Freetime
+@testable import StyledTextKit
+@testable import GitHubAPI
+
+class SortingMentionedByDayTests: XCTestCase {
+    func testSortingMentionedByDay() {
+        // MARK: - given
+        let models = 5
+        let parsed = InboxDashboardModelBuilder.provide(number: models)
+        let mentioned = V3IssuesRequest.FilterType.mentioned
+        let created = V3IssuesRequest.FilterType.created
+        let assigned = V3IssuesRequest.FilterType.assigned
+        // MARK: - when
+        let mentionedSorted = parsed.sortedMentionedByDate(filter: mentioned)
+        let createdSorted = parsed.sortedMentionedByDate(filter: created)
+        let assignedSorted = parsed.sortedMentionedByDate(filter: assigned)
+        // MARK: - then
+        XCTAssert(zip(mentionedSorted,
+            mentionedSorted.sorted(by: {$0.date > $1.date}))
+            .map { $0.date == $1.date }
+            .reduce(true) { $0 && $1 }, "Mentioned is not sorted by day")
```

Comment:
You can use [SE-0207](https://github.com/apple/swift-evolution/blob/master/proposals/0207-containsOnly.md)'s [`allSatisfy`](https://developer.apple.com/documentation/swift/array/2994715-allsatisfy) here ;)

Reply:
Wow 😲 
I did not know about it. Fixed it.